pylintrc

```diff
@@ -92,7 +97,8 @@ disable=
     useless-super-delegation,  # new in pylint 1.7
     redefined-argument-from-local,  # new in pylint 1.7
     consider-merging-isinstance,  # new in pylint 1.7
-
+    assignment-from-no-return,  # new in pylint 2.0
+    consider-using-get,  # new in pylint 2.0
```

Comment:
Are you planning to address these issues later? ``assignment-from-no-return`` seems easy to fix, add ``return None``.

Reply:
Pylint is complaining that `collections.deque.popleft()` is violating `assignment-from-no-return`, so I think this can be revisited in the future, but I can fix methods that we control right now.

Reply:
After adding `return None` to some method to fix `assignment-from-no-return`:
```
[E1128(assignment-from-none)] Assigning to function call which only returns None)
[R1711(useless-return)] Useless return at end of function or method)
```